gui/system/forms.py

```diff
@@ -2606,9 +2637,22 @@ class CertificateAuthorityCreateIntermediateForm(MiddlewareModelForm, ModelForm)
         required=True,
         help_text=models.CertificateAuthority._meta.get_field('cert_name').help_text
     )
+    cert_key_type = forms.ChoiceField(
+        label=_('Key Type'),
+        help_text=_('Please select a key type to proceed with'),
+        required=True,
+        choices=()
+    )
+    cert_ec_curve = forms.ChoiceField(
+        label=_('EC Key Curve'),
+        help_text=_('Please select a curve to use to generate EC key'),
```

Comment:
As above.  If you can provide more context on these, we can help write text that helps the user make the correct choice.